src/microbe_stage/Microbe.cs

```diff
@@ -455,6 +460,10 @@ public override void _Process(float delta)
                     movementAudio.Play();
             }
         }
+        else
+        {
+            Colony?.Master?.AddMovementForce(queuedMovementForce);
```

Comment:
Should it maybe be assumed that if ColonyParent is not null, then Colony can't be null either? So the first ? could be removed here very safely.
Also doesn't this also have a multi frame dependency? As in if the master microbe runs its _Process first this movement force will only apply on the next frame. This dependency over multiple _Process calls should be documented here.

Reply:
I think here the formatter didnt let me do that but ill check.

Reply:
If that's the case I think `else if(Colony != null)` with a comment explaining why it is like that would be better than this code.

Reply:
huh? why? why notleave it compressed like that? standard null propagation

Reply:
In the If it's already checked that the cell is a part of a colony, so Colony variable should not be null, also (I didn't design the colony system, but I think this is true) the Master in a colony should never be null. A colony can't exist without a master. So it doesn't make sense to do null checking on them here, if the linter truly complains here, I'll accept null checks, but logically they don't make any sense here.

Reply:
The linter complains
![image](https://user-images.githubusercontent.com/11059623/149359756-4756f700-5756-4ae9-80e3-9be857294111.png)

Reply:
the inspect_results says nothing more usually then what is printed.
<Issue TypeId="PossibleNullReferenceException" File="src\microbe_stage\Microbe.cs" Offset="14746-14752" Line="466" Message="Possible 'System.NullReferenceException'" />

Reply:
![image](https://user-images.githubusercontent.com/11059623/149360710-2f649bde-f04a-4c6e-9b2a-efd815298018.png)

Reply:
The offset points to the exact variable that can be null. At least in emacs you can go to the n:th character in a file so you can use those.